docs/file_system.md

```diff
@@ -25,24 +25,27 @@ The file descriptor API may be found [here](http://angr.io/api-doc/angr.html#ang
 Okay okay!!
 
 To create a SimFile, you should just create an instance of the class you want to use.
-Refer to the [api docs](http://angr.io/api-doc/angr.html#module-angr.storage.file) for the full instructions.
+Refer to the [API docs](http://angr.io/api-doc/angr.html#module-angr.storage.file) for the full instructions.
 
-Let's go through a few illustrative examples.
+Let's go through a few illustrative examples, which cover how you can work with a concrete file, a symbolic file, a file with mixed concrete and symbolic content, or streams.
 
 #### Example 1: Create a file with concrete content
 
 ```python
+>>> import angr
 >>> simfile = angr.SimFile('myconcretefile', content='hello world!\n')
 ```
 
-Here's a nuance - you can't use simfiles without a state attached, because reasons.
-You'll never have to do this in a real scenario (this operation happens automatically when you pass a SimFile into a constructor or the filesystem) but let's mock it up:
```

Comment:
This should in fact be a constructor *or* the filesystem - I was referring to `state.fs.insert` or whatever.